notebooks/3_run_model.ipynb

```diff
@@ -384,11 +366,42 @@
   {
    "cell_type": "code",
    "execution_count": null,
+   "source": [
+    "alpha = 0.6\n",
+    "beta = 0.2\n",
+    "mu = 0\n",
+    "lamda_c_list = [200]\n",
+    "phi = 1\n",
+    "w_phi = 1\n",
+    "start_year = 2000\n",
+    "stop_year = None\n",
+    "random_seed = None\n",
+    "\n",
+    "# Transmission lag type to include (e.g., None, static, stochastic)\n",
+    "transmission_lag_type = \"stochastic\" \n",
+    "\n",
+    "# If transmission lag type is static, set time_infect as number of \n",
+    "# years to delay country from becoming an origin. If lag type is\n",
+    "# none or stochastic, set to None\n",
+    "time_infect = None \n",
+    "time_infect_units = \"year\"\n",
+    "\n",
+    "# If transmission lag type is stochastic, set values to generate\n",
+    "# a gamma distribution. If lag type is none or static, ste to None. \n",
+    "gamma_shape = 4\n",
+    "gamma_scale = 1"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
    "source": [
     "# Name model results: simultation name, additional description (i.e., parameter value\n",
     "# or scenario tested), and run number to identify stochastic run. \n",
-    "sim_name = f'slf_test'\n",
-    "add_descript = 'inflation_adjusted'\n",
+    "sim_name = f'tuta_test'\n",
```

Comment:
I opted to restore the lines I had edited for tuta back to what is in the main repo for SLF. I'm not sure of the best way to avoid leaving traces of case studies in these notebooks. It may not be a problem, but it might cause some headaches if we want to be consistent about which case study is used in the repo version, as in this example. Maybe saving a config file with these case study specific values and then copy and paste from that to update values in the notebooks is good enough? Might be worth thinking through what the most flexible solution is.

Reply:
Agreed - perhaps for any published case studies we can have a separate repo to retain the output, but for the main repo a config/yaml file is a good solution.